dasharo-stability/capsule-update.robot

```diff
@@ -53,7 +55,12 @@ CUP001.001 Capsule Update With Wrong Keys
 
     Enter Capsule Testing Folder
     ${out}=    Execute UEFI Shell Command    CapsuleApp.efi -S
-    Should Contain    ${out}    Capsule Status: Security Violation
+    ${out}=    Replace String    ${out}    \n    \\n
+    IF    '${EXPECTED_STATUS_1}' in '${out}' or '${EXPECTED_STATUS_2}' in '${out}'
```

Comment:
Why do this? Wrong keys should not cause `Not Ready`.

Reply:
@SergiiDmytruk This is why we all need to use same binaries and capsules. Preparing them have quite a few arguments which can cause various differences in the output and different behavior during tests.
Could I ask you to prepare a correct set of rom and cap files for me and @philipandag ??

Reply:
I'm not sure how you can get `Not Ready` here.  I'm using your script to create them.  I can make/upload binaries, but we should have a procedure that anyone can follow.  All I do is:
 * build unmodified firmware for initial flashing
 * enable serial there
 * update version and rebuild
 * enable serial there
 * make a capsule out of updated version (`./capsule.sh make -t test-keys/TestRoot.pub.pem -o test-keys/TestSub.pub.pem -s test-keys/TestCert.pem` with test keys coming from `Pkcs7Sign` in EDK)
 * use that capsule to generate test capsules

Reply:
@SergiiDmytruk please create binaries and upload them as we need to perform tests. We will focus on creating easy to follow procedure when we will be sure that tests work as intended.

Reply:
[osfv-capsules-ms7d25.zip](https://github.com/user-attachments/files/16929298/osfv-capsules-ms7d25.zip)
[osfv-capsules-ms7d25-ddr4.zip](https://github.com/user-attachments/files/16929308/osfv-capsules-ms7d25-ddr4.zip)
[osfv-capsules-ms7e06.zip](https://github.com/user-attachments/files/16929312/osfv-capsules-ms7e06.zip)
[osfv-capsules-qemu-q35.zip](https://github.com/user-attachments/files/16929315/osfv-capsules-qemu-q35.zip)